packages/SidePanel/src/hooks/useEscapeKey.js

```diff
@@ -1,19 +1,23 @@
 import React from "react";
```

Comment:
maybe we can unified this component on the helpers and reuse it in modal, sidepanel and takeover component?